### DISCUSSION TITLE
Multi Author Support

### DISCUSSION THREAD

**Discussion by satrox28** [thuliteio/doks#1265] [General]
Hi, 
Thanks for creating this great theme.

Do doks support multi author? and I noticed that Hugo support it but would like to get a confirmation and if possible an example to share.

Currently I can see the contributors page but authors page not listing any authors and am trying to customize it.

**Comment by satrox28** [comment]
@h-enk 

Can you check and let me know, thanks :)

**Reply by h-enk** [reply]
Yes, Doks supports multiple authors (I named it contributors, but you could easily change that).

For example, add to the frontmatter in `content/blog/example/index.md`:

```md
contributors: ["Henk Verlinde", "Sathish Kumar", "Anders Norén"]
```

![2024-08-06_14-09-04](https://github.com/user-attachments/assets/ef30e53f-fa0b-449f-8e65-b7026a05af6b)

**Reply by satrox28** [reply]
got it but I was looking for a dedicated Authors page, some thing like below with an Intro and his/her posts.

https://www.smashingmagazine.com/author/vitaly-friedman/

**Reply by h-enk** [reply]
Create the following pages (in the frontmatter set `draft: false`):

```bash
npm run create contributors/_index.md
```

```bash
npm run create contributors/sathish-kumar/_index.md
```

In the latter add your content (I added "Lorem ipsum") — `http://localhost:1313/contributors/sathish-kumar/`:

![2024-08-06_19-03-23](https://github.com/user-attachments/assets/53baba0a-cd02-42bb-a615-48e648b83f75)

Need more customization? Copy `node_modules/@hyas/doks-core/layouts/_default/term.html` and paste as `layouts/_default/term.html`. See also:  

- [Henk Verlinde](https://getdoks.org/contributors/henk-verlinde/)
- [henk-verlinde](https://github.com/gethyas/getdoks.org/blob/main/content/contributors/henk-verlinde/_index.md)

**Reply by satrox28** [reply]
Thanks, will give it a try. Is it possible to show all authors name in a page.

**Reply by satrox28** [reply]
got it from this url

http://localhost:1313/contributors/

Thanks :)
